ThirdPartyAdapters/chartboost/chartboost/src/main/java/com/google/ads/mediation/chartboost/ChartboostAdapterUtils.java

```diff
@@ -42,15 +44,18 @@
      * bundles provided.
      */
     static ChartboostParams createChartboostParams(Bundle serverParameters, Bundle networkExtras) {
+        String adLocation = "";
         ChartboostParams params = new ChartboostParams();
-        String appId = serverParameters.getString(KEY_APP_ID);
-        String appSignature = serverParameters.getString(KEY_APP_SIGNATURE);
-        if (appId != null && appSignature != null) {
-            params.setAppId(appId.trim());
-            params.setAppSignature(appSignature.trim());
+        if(serverParameters != null) {
+            String appId = serverParameters.getString(KEY_APP_ID);
+            String appSignature = serverParameters.getString(KEY_APP_SIGNATURE);
+            if (appId != null && appSignature != null) {
+                params.setAppId(appId.trim());
+                params.setAppSignature(appSignature.trim());
+            }
+            adLocation = serverParameters.getString(KEY_AD_LOCATION);
         }
 
-        String adLocation = serverParameters.getString(KEY_AD_LOCATION);
         if (!isValidParam(adLocation)) {
```

Comment:
Just to be sure, `isValidParam` returns false if the string is either null or empty right?